### PR TITLE
Create REVIEWME.txt

### PR DESCRIPTION
**Pull request by MichaelWong1024**

### CODE REVIEW

File: REVIEWME.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Please review me!
```
Comment (SonyaInSiberia) [inline REVIEWME.txt:1]: The implementation of comb filter and delay buffer logic looks pretty clean overall. One thing to notice and be careful about is that the read audio from hound has dimension [channel_size][sample_length] but the handling took it reversed, it might cause some problems later on. 
I like how the edge cases are considered and handled in main with proper error messages reported. The test cases looks pretty well implemented on generating different kinds of output (which confused me a lot when I was implementing them).

Comment (SonyaInSiberia) [reply REVIEWME.txt:1]: One more thing I think you need to be more careful about is the size of delay buffer and input, it seems like you initialize one for every channel, it is ok in your implementation since you did not divide the input into multiple windows/chunks (which is a pretty typical process for audio processing) and processed it in one loop, but when it comes to industrial standards, you might want to consider handle the continuous delay between each chunk with proper splitting methods since processing the whole audio at once might not be practical for a lot of its real-life scenarios.

**Comment by venkatKrishnan86** [comment]
Your code works correctly, and gives the correct outputs, but there seems to be an efficiency problem. A three second stereo audio did not take much time, but when I tried running a 25 second 9 channel Spatial Audio through it with a gain of 0.8 and delay of 0.1 seconds, and in release mode it took 59 seconds to complete, and longer the audio, larger is the computation time, which is not good for real time computing. On average, it should not take more than 0.2-0.5 seconds for the spatial example. I think you will need to resolve this by checking your buffer implementation. I suspect because you have nested for loops, if the buffers are not managed properly, it can lead to $O(n^2)$ complexity, which will keep increasing as the size increases at a quadratic rate.
